requirements.txt

```
@@ -1,7 +1,7 @@
-numpy~=1.21.2
-torch~=1.9.0
-torchvision~=0.10.0
-Pillow~=7.2.0
-lightly~=1.1.18
+torch~=2.0.0
+torchvision~=0.15.0
+Pillow~=9.0.0
+lightly~=1.4.6
```

```suggestion
lightly>=1.4.6
```

Not sure if this is a good idea but might keep the tutorial less of out of date :)

---

It should be fine since we probably won't update the client methods that much.

---

Yes but as soon as we increase config version things might start to break because either API or worker won't support it anymore.

---

In that case, we will need to update many tutorials anyway. It should be fine.